src/components/ui/Mark.tsx

```diff
@@ -23,26 +23,37 @@ const questionStyles = (small?: boolean, disabled?: boolean) =>
     cursor(disabled ? undefined : 'cursor-pointer')
   )
 
-const borderWrapper = (small?: boolean) =>
+const borderWrapper = (small?: boolean, medium?: boolean) =>
   classnames(
     boxSizing('box-content'),
     borderRadius('rounded-full'),
     borderWidth('border'),
     borderColor('border-current'),
-    width(small ? 'w-4' : 'w-6'),
-    height(small ? 'h-4' : 'h-6')
+    width({ 'w-4': small, 'w-5': medium, 'w-6': !small && !medium }),
```

Comment:
And I think this logic can be simplified, let's think how we can do this

Reply:
I tried to simplify it, have no idea how to simplify it more